lib/getstats.js

```diff
@@ -290,15 +292,17 @@ function getTracks(peerConnection, kind, localOrRemote) {
  */
 function getTrackStats(peerConnection, track, options) {
   options = options || {};
+  var browser = guessBrowser();
 
-  if (typeof options.testForChrome !== 'undefined' ||
-    typeof webkitRTCPeerConnection !== 'undefined') {
+  if (browser === 'chrome' || typeof options.testForChrome !== 'undefined') {
     return chromeGetTrackStats(peerConnection, track);
   }
-  if (typeof options.testForFirefox  !== 'undefined' ||
-    typeof mozRTCPeerConnection !== 'undefined') {
+  if (browser === 'firefox' || typeof options.testForFirefox  !== 'undefined') {
     return firefoxGetTrackStats(peerConnection, track);
   }
+  if (browser === 'edge') {
```

Comment:
@syerrapragada How do you unit test for edge here without an `options.testForEdge` flag?